Doc/library/sqlite3.rst

```diff
@@ -1320,27 +1321,11 @@ Connection objects
       a :class:`Cursor` object and the raw row results as a :class:`tuple`,
       and returns a custom object representing an SQLite row.
 
-      Example:
-
-      .. doctest::
-
-         >>> def dict_factory(cursor, row):
-         ...     col_names = [col[0] for col in cursor.description]
-         ...     return {key: value for key, value in zip(col_names, row)}
-         >>> con = sqlite3.connect(":memory:")
-         >>> con.row_factory = dict_factory
-         >>> for row in con.execute("SELECT 1 AS a, 2 AS b"):
-         ...     print(row)
-         {'a': 1, 'b': 2}
-
       If returning a tuple doesn't suffice and you want name-based access to
       columns, you should consider setting :attr:`row_factory` to the
-      highly optimized :class:`sqlite3.Row` type. :class:`Row` provides both
-      index-based and case-insensitive name-based access to columns with almost no
-      memory overhead. It will probably be better than your own custom
-      dictionary-based approach or even a db_row based solution.
+      highly optimized :class:`sqlite3.Row`
```

Comment:
Something is off here. Let's try this.

```suggestion
      highly optimized :class:`sqlite3.Row` row factory.
```

Reply:
What about `class`?  The doc said `type` before, and this sentence is already talking about row factories.

Reply:
Yeah, that may be better.